sympy/physics/_biomechanics/curve.py

```diff
@@ -1738,3 +1742,15 @@ def _latex(self, printer):
         fv_M = self.args[0]
         _fv_M = printer._print(fv_M)
         return r'\left( \operatorname{fv}^M \right)^{-1} \left( %s \right)' % _fv_M
+
+
+@dataclass(frozen=True)
+class CharacteristicCurveCollection:
+    """Simple data container to group together related characteristic curves."""
+    tendon_force_length: CharacteristicCurveFunction
+    tendon_force_length_inverse: CharacteristicCurveFunction
+    fiber_force_length_passive: CharacteristicCurveFunction
+    fiber_force_length_passive_inverse: CharacteristicCurveFunction
+    fiber_force_length_active: CharacteristicCurveFunction
+    fiber_force_velocity: CharacteristicCurveFunction
+    fiber_force_velocity_inverse: CharacteristicCurveFunction
```

Comment:
These were not curves before, right? So does this now mean we can (in the future) control the display of the muscle-tendon force, for example?

Reply:
This was required so that the musculotendon (`MusculotendonDeGroote2016`) class can have access to the curve functions (these) so that it correctly deal with the `with_defaults` constructor. The `force` attribute is just a SymPy expression, so I'm not sure how we could simplify the display of that directly. We could, however, implement `__str__` on all classes in `sympy.physics.biomechanics` in a way that this produces a nice user-friendly display.

Reply:
Got it, my confusion.